.gitignore

```diff
@@ -12,3 +12,5 @@ examples/Outputs/
 JuliaSysimage.dll
 
 docs/build/
+
+/test/jl*
```

Comment:
This will ignore any file starting with `jl` in a directory called `test` at the root of the file system. I don't think this works for Windows but I'm also not sure what you're trying to ignore here?

Reply:
When I run tests, they generate a large number of files starting with jl in the test folder. I assumed this was to do with the new tests for running scenarios

Reply:
Yes, but don't they get auto-cleared?

I think my point above still stands though, `/test` points to the root of the file system, it's not local to the project directory and on Windows systems it's a nonsensical file path (I think) so gitignore is ignoring a non-existent path. 

See `docs/build/` above this entry - there's no leading slash.

Reply:
How do you get them to auto-clear?

Reply:
It doesn't create files starting with `jl`, but it does create a folder with `jl` - you were correct in that these are temporary directories to store the example run that occurs as part of a test.

The directory should be automatically cleared/deleted after tests finish running (and this happens for me, I just checked).

The failing test is new though, doesn't happen on `Main`.

Reply:
The failing test also doesn't happen in `consistent-cover`, but a different error to do with `proportional_adjustment!()` does happen 🤔 

```julia
proportional adjustment: Error During Test at C:\Users\tiwanaga\projects\ADRIA.jl\test\growth.jl:5
  Got exception outside of a @test
  MethodError: no method matching proportional_adjustment!(::Matrix{Float64}, ::Vector{Float64}, ::Vector{Float64})

  Closest candidates are:
    proportional_adjustment!(::Union{SubArray{T}, Matrix{T}}, ::Vector{T}) where T<:Float64
     @ ADRIA C:\Users\tiwanaga\projects\ADRIA.jl\src\ecosystem\corals\growth.jl:43
    proportional_adjustment!(::Union{SubArray{T}, Matrix{T}}) where T<:Float64
     @ ADRIA C:\Users\tiwanaga\projects\ADRIA.jl\src\ecosystem\corals\growth.jl:66
```

Reply:
hmm weird I'll remove the line and try rebasing and see if the test still fails

Reply:
ok I think maybe something weird happened when I rebased with consistent-cover the first time, I copied and pasted the growth.jl test from consistent-cover and now I'm only getting the proportional adjustment error.

Reply:
Did you compare this revision with the previous one to see what the difference was and what was causing the issue?

Reply:
Yeah so I just looked into this further. I don't think it's a rebase issue now, the ODE test error is still there if I comment out the proportional cover test. The proportional cover test error is intoduced because tmp is added as an additional argument in `proportional_adjustment!` on line 11 of the growth.jl test in consistent-cover. If I remove the `tmp` argument the test passes (which is how it was in my branch before I compared with consistent-cover).

Reply:
I'll look into a fix for the the ODE issue myself (as you suggested just now on #551.